invenio_search/contrib/records.py

```diff
@@ -27,8 +27,8 @@
 import warnings
 
 from flask_sqlalchemy import models_committed
-from invenio_records.models import RecordMetadata
 
+from invenio_records.models import RecordMetadata
```

Comment:
I don't think this import sorting is correct (probably ``invenio_records``  is missing in ``known_third_party`` in ``.editorconfig``)